installer/templates/new/widgets/number/number.js

```diff
@@ -36,15 +36,24 @@ Widget.mount(class Number extends Widget {
       return '';
     }
   }
+  changeRate() {
+    if (this.props.changerate == "off") {
```

Comment:
I'd prefer an early return here.

```javascript
if (is-it-disabled?) { return null };

code
```

I'm also a bit concerned about the equality check with `"off"`.
If the user sets the prop to `false` it won't work. I think we currently don't have a convention over toggling widget features on/off. Can't we change it to `this.props.changerate === false` ?